### DISCUSSION TITLE
Möglichkeiten Display

### DISCUSSION THREAD

**Discussion by f1nch87** [solectrus/solectrus#3498] [Show and tell]
Guten Tag,

aktuell bin ich am überlegen wie ich am schönsten ein Display ausschließlich für Solectrus im Haus installiere?

Habe noch ein altes Tablet was theoretisch dafür in Kombination mit dem Kiosk Browser genutzt werden könnte und dazu eine Rahmen fräsen, drucken oder so.

Auch habe ich noch Raspi's rumliegen, allerdings kein großes Display mit Touch und außerdem wüsste ich nicht wie ich das einrichte, dass es mir ausschließlich Solectrus anzeigt und auch in den Screensaver geht oder so z.B.

Vlt gibt es hier Leute die so etwas schon realisiert haben und Erfahrungen, Bilder, Tipps teilen könnten.

Kostengünstig wäre schön.

Danke
Gruß

**Comment by ledermann** [comment]
Von @nk-henning gibt es diesen - schon etwas älteren - Blog-Artikel:
https://mitkaracho.de/posts/2023/solectrus-mit-senec/

Weitere Berichte mit Fotos sind hier sehr gerne gesehen - je exotischer, desto besser :)
Möchte noch jemand seine SOLECTRUS-Installation vorstellen?

**Reply by f1nch87** [reply]
So hatte endlich mal Zeit das Dashboard Projekt umzusetzen.
Die blaue 3D gedruckte Front wird noch aus nem hübschen Holz gefräst falls ich Zeit finde die CNC wieder in Gang zu setzen.

Gruß und danke fürs schönste PV Dashboard![IMG_20250423_102326.jpg](https://github.com/user-attachments/assets/add6cf77-ff20-4c4e-8519-44ea91a22d56)

**Reply by ledermann** [reply]
Tolle Arbeit, danke für das Foto!
